### PR TITLE
refactor(CLOUD-234): moved the resource code to the appropriate files

### PR DESCRIPTION
**Pull request by anaxaim**

### CODE REVIEW

File: edgecenter/utils_instance.go
Comment (DamirZinatullin) [inline edgecenter/utils_instance.go:1]: Perhaps the "utils" functions should be placed in a separate module in accordance with https://github.com/golang-standards/project-layout/blob/master/README.md#go-directories ?

File: edgecenter/utils_loadbalancer.go [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+	"github.com/Edge-Center/edgecentercloud-go/edgecenter/loadbalancer/v1/lbpools"
+	"github.com/Edge-Center/edgecentercloud-go/edgecenter/loadbalancer/v1/listeners"
+	typesLb "github.com/Edge-Center/edgecentercloud-go/edgecenter/loadbalancer/v1/types"
+)
+
+// ImportStringParserExtended parses a string containing project ID, region ID, and two other fields,
+// and returns them as separate values along with any error encountered.
+func ImportStringParserExtended(infoStr string) (int, int, string, string, error) {
+	log.Printf("[DEBUG] Input id string: %s", infoStr)
```
Comment (DamirZinatullin) [inline edgecenter/utils_loadbalancer.go:19]: Why we don't use named return values?

**Review by DamirZinatullin — COMMENTED** [review]
Viewed

File: edgecenter/utils_instance.go
Comment (anaxaim) [reply edgecenter/utils_instance.go:1]: postpone until the task https://tracker.yandex.ru/CLOUD-355 completed

**Review by DamirZinatullin — APPROVED** [review]
Ok

**Pull request merged**